loyaltyCards.sh

```diff
@@ -1,2 +1,5 @@
 #!/bin/sh
-python3 ~/.local/share/loyaltyCards/loyaltyCards.py
```

Comment:
Why changing the approach? Can you explain me this change please? Thank you!

Reply:
The new file should read 
`#!/bin/sh`
` `
`cd ~/.local/share/loyaltyCards`
`python3 loyaltyCards.py &`
`exit`
This starts loyaltyCards in a new thread and the exits so that the shell script does not keep running until loyaltyCards exist